code/modules/admin/verbs/put_mob_into_marked.dm

```diff
@@ -0,0 +1,27 @@
+/client/proc/put_mob_into_marked(mob/target_mob in GLOB.mob_list)
+	set category = "Fun"
+	set desc = "(atom path) Puts a mob into the currently marked obj"
+	set name = "Put Mob Into Marked"
+
+
+	if(!check_rights(R_ADMIN))
+		return
+	if(!holder)
+		return
+	var/obj/t = holder.marked_datum
+
+	if(!t)
+		return alert(usr, "You need to mark an object first!", null, null, null, null)
+	if(!t.loc)
+		return alert(usr, "The object you marked ([t]) doesn't have a location!", null, null, null, null)
+
+	t.contents += target_mob
+	t.vis_contents += target_mob
+
+	var/obj/cur = t
```

Comment:
why is this typecasted to obj

Reply:
I don't know either, which is why the typecast was removed in an as-of-yet unpushed version